ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -413,6 +377,88 @@ public struct Avatar: View, ConfigurableTokenizedControl {
         }
     }
 
+    /// Handles calculating colors for Avatar foreground and background.
+    private struct CalculatedColors {
```

Comment:
can we move it to a separate file so that in case we ever need to expose this to clients we can make the struct public easily?

Reply:
I know we discussed that a bit this morning, but after reviewing the code I believe that customization will be best served by the existing `backgroundColor` and `foregroundColor` properties on the `AvatarState` object, and this makes sense to keep private. Thoughts?